packages/expo-modules-autolinking/scripts/ios/autolinking_manager.rb

```diff
@@ -216,7 +216,8 @@ class AutolinkingManager
         'node',
         '--no-warnings',
         '--eval',
-        'require(require.resolve(\'expo-modules-autolinking\', { paths: [\'' +  __dir__ + '\'] }))(process.argv.slice(1))',
+        'require(\'expo/bin/autolinking\')',
+        'expo-modules-autolinking',
```

Comment:
This technically changes its behaviour. Technically, we'd like to just avoid the self-require here, for our monorepo. However, there's little point in this one script resolving from "itself" rather than re-entering `expo` while the Android side doesn't bother doing this